themes/default.sh

```diff
@@ -70,6 +70,7 @@ if [ -z $TMUX_POWERLINE_RIGHT_STATUS_SEGMENTS ]; then
 	TMUX_POWERLINE_RIGHT_STATUS_SEGMENTS=(
 		#"earthquake 3 0" \
 		"pwd 89 211" \
+		"tun0 13 255" \
```

Comment:
As it's not common to have a tun interface, let's not add this to the default theme. It's easy to make your own theme outside that includes it :)

Reply:
Gotcha! Thanks for you time!